### PR TITLE
Add Reducers and Actions

### PR DESCRIPTION
**Pull request by demesameneshoa**
## In this PR I executed 🔢 
- [x] Install Redux toolkit
- [x] Create Store.js
- [x] Define book slice and category slice
- [x] Create add book and remove book reducers 
- [x] Create category reducer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,7 +123,7 @@ npm start
 
 ## 🔭 Future Features <a name="future-features"></a>
```
Comment (Ismailco) [inline README.md:112]: - [x] Kindly consider removing the `Live Demo` section altogether until you deploy the website

Comment (Ismailco) [inline README.md:112]: - [x] The issues link is broken kindly make sure to fix it so all the likes in your README are working properly

File: src/redux/categories/categoriesSlice.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState: { categories: [], status: 'Under construction' },
```
Comment (Ismailco) [inline src/redux/categories/categoriesSlice.js]: - [x] The category initialState should contain an empty array, kindly consider implementing this

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @demesameneshoa, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @demesameneshoa , 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Greate readme file
✅ ... Redux initialize

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**